Python Tutorial Exercises(Week 3)/#7.py

```diff
@@ -0,0 +1,66 @@
+'''
+Solve Week 2 exercises using while loops instead of for loops
+'''
+#1
+x = [["this", "is", "a", "nested"], ["list", "that"], ["will", "be", "printed", "out"]]
+def nest_list(x):
+    
+    new_list = []
+    z = 0
+    
+    while z < len(x):
+        if isinstance(x[z], list):
+            new_list.extend(nest_list(x[z])) 
+
+        else:
+            new_list.append(x[z])
+        z += 1
+
+    return new_list
+
+i = 0
+while i < len(nest_list(x)):
+    print(nest_list(x)[i])
```

Comment:
here you're calling the same function `nest_list` multiple times, which the result of that function does not change. Should initialise a new variable to hold the result of the unwrapped list first for efficiency sake

Reply:
How to do that?

Reply:
The same way you store any value in a variable `x = 0`
https://quick-adviser.com/how-do-you-store-return-value-in-a-variable/